prowler/providers/azure/services/containerregistry/containerregistry_service.py

```diff
@@ -37,8 +37,10 @@ def _get_container_registries(self):
                                 resource_group=resource_group,
                                 sku=getattr(registry.sku, "name", ""),
                                 login_server=getattr(registry, "login_server", ""),
-                                public_network_access=getattr(
-                                    registry, "public_network_access", ""
+                                public_network_access=(
+                                    False
+                                    if registry.public_network_access == "Disabled"
```

Comment:
```suggestion
                                    if getattr(registry, "public_network_access" "Enabled") == "Disabled"
```
Please add this, as according to [Microsoft documentation](https://learn.microsoft.com/en-us/python/api/azure-mgmt-containerregistry/azure.mgmt.containerregistry.v2023_11_01_preview.models.registry?view=azure-python) `public_network_access` is not mandatory and could be None.